source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -1002,14 +1037,9 @@ bool COpenGLDriver::genericDriverInit()
 	// adjust flat coloring scheme to DirectX version
 	///extGlProvokingVertex(GL_FIRST_VERTEX_CONVENTION_EXT);
 
-	// create material renderers
-	createMaterialRenderers();
-
-	// set the renderstates
-	setRenderStates3DMode();
-
 	// We need to reset once more at the beginning of the first rendering.
 	// This fixes problems with intermediate changes to the material during texture load.
+    //TODO hm? should i worry about that?
```

Comment:
best try and nuke it... AFAIK its irrlicht's poor counterpart to our state flush

Reply:
resolved